installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -68,10 +70,10 @@ podman pull registry.redhat.io/rhel9/postgresql-15:latest
 ----
 --
 
-. Push both images to the internal OCP registry.
+. Push both images to the internal OpenShift image registry.
```

Comment:
```suggestion
. Push both images to the internal {ocp-short} image registry.
```

Reply:
I intentionally left the attribute out because the OCP docs use _OpenShift image registry_ but I will accept your suggestion because I'm not sure if the OCP docs are actually following brand standards and I prefer to use the attribute for all instances of _OpenShift_.